core/node/proof_data_handler/Cargo.toml

```diff
@@ -10,13 +10,24 @@ keywords.workspace = true
 categories.workspace = true
 
 [dependencies]
+vm_utils.workspace = true
 zksync_config.workspace = true
 zksync_dal.workspace = true
 zksync_object_store.workspace = true
 zksync_prover_interface.workspace = true
+zksync_tee_verifier.workspace = true
 zksync_types.workspace = true
-
-tracing.workspace = true
+zksync_utils.workspace = true
```

Comment:
Same

Reply:
Missed it. [Fixed](https://github.com/matter-labs/zksync-era/pull/1993/commits/0114a3cdf5a36650d4cc26cbf59e66c5b3e5014b). ✅